server/helpers/email/mailer.js

```diff
@@ -0,0 +1,12 @@
+import nodemailer from 'nodemailer';
+
+let transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
+
+export default transporter;
```

Comment:
I've moved the nodemailer implementation into its own separate module file, as I'm sure we're going to want to reference and use it elsewhere without having to duplicate the code. 

Also, I think we'll want to bring in an email templating system for nicer emails, and we can do that all in one place here.